src/Pagination/Pagination.tsx

```diff
@@ -157,7 +156,7 @@ const PaginationContainer = styled.nav`
 `
 
 export type PaginationProps = {
-  theme: object
+  theme?: object
```

Comment:
Is it still necessary to directly pass `theme` down to `Box` below on line 192? Shouldn't `Box` get `theme` as a prop if it's wrapped by a theme provider?

Reply:
Since we still support the `theme` prop in all components, we still want the `theme` prop to override the theme from the ThemeProvider. This PR is about making sure that we're not accidentally overriding the ThemeProvider out of the box.

I'm open to removing the theme prop entirely and make people use a ThemeProvider to change the theme but that's a much bigger change that's out of scope for this PR.